### PR TITLE
Codellama-13b python,34b-Python,7b-Python-hfDesc

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
Added Codellama-13b python,34b-Python,7b-Python-hf Desc

# Model Evaluation samples

Task| Use case| Dataset| Python sample (Notebook)| CLI with YAML
|--|--|--|--|--|
Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>

## **Inference samples**
Inference type|Python sample (Notebook)|CLI with YAML
|--|--|--|
Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon

### CODE REVIEW

File: assets/models/system/CodeLlama-13b-Python-hf/description.md [hunk 1]
```diff
@@ -62,6 +62,20 @@ All experiments reported here and the released models have been trained and fine
...
+Task| Use case| Dataset| Python sample (Notebook)| CLI with YAML
+|--|--|--|--|--|
+Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>
+
+
+# **Inference samples**
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation" target="_blank">text-generation-online-endpoint.sh</a>
```
Comment (novaturient95) [inline assets/models/system/CodeLlama-13b-Python-hf/description.md:76]: this is not the correct notebook. Could you update link to this one: https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/inference/text-generation/text-generation-online-endpoint-dolly.ipynb instead

File: assets/models/system/CodeLlama-34b-Python-hf/description.md [hunk 2]
```diff
@@ -62,6 +62,20 @@ All experiments reported here and the released models have been trained and fine
...
+Task| Use case| Dataset| Python sample (Notebook)| CLI with YAML
+|--|--|--|--|--|
+Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>
+
+
+# **Inference samples**
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation" target="_blank">text-generation-online-endpoint.sh</a>
```
Comment (novaturient95) [inline assets/models/system/CodeLlama-34b-Python-hf/description.md:76]: same as above. Please update the the notebook link

File: assets/models/system/CodeLlama-13b-Python-hf/description.md (hunk 1 above)
Comment (KonakanchiSwathi) [reply assets/models/system/CodeLlama-13b-Python-hf/description.md]: Updated inference links to 
## **Inference samples**
Inference type|Python sample (Notebook)|CLI with YAML
|--|--|--|
Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon

File: assets/models/system/CodeLlama-34b-Python-hf/description.md (hunk 2 above)
Comment (KonakanchiSwathi) [reply assets/models/system/CodeLlama-34b-Python-hf/description.md]: updated inference links
## **Inference samples**
Inference type|Python sample (Notebook)|CLI with YAML
|--|--|--|
Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon

File: assets/models/system/CodeLlama-7b-Python-hf/description.md [hunk 3]
```diff
@@ -69,13 +69,13 @@ Task| Use case| Dataset| Python sample (Notebook)| CLI with YAML
 Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>
 
 
-# **Inference samples**
-
+## **Inference samples**
```
Comment (novaturient95) [inline assets/models/system/CodeLlama-7b-Python-hf/description.md:72]: use single `#`

Comment (KonakanchiSwathi) [reply assets/models/system/CodeLlama-7b-Python-hf/description.md]: Remove one #

**Review by novaturient95 — DISMISSED** [review]

File: assets/models/system/CodeLlama-7b-Python-hf/description.md [hunk 4]
```diff
@@ -62,6 +62,19 @@ All experiments reported here and the released models have been trained and fine
 
 See evaluations for the main models and detailed ablations in Section 3 and safety evaluations in Section 4 of the research paper.
 
+# Model Evaluation samples
```
Comment (novaturient95) [inline assets/models/system/CodeLlama-7b-Python-hf/description.md:65]: the place for samples is not correct. Can you please update

File: assets/models/system/CodeLlama-13b-Python-hf/description.md [hunk 5]
```diff
@@ -62,6 +62,19 @@ All experiments reported here and the released models have been trained and fine
 
 See evaluations for the main models and detailed ablations in Section 3 and safety evaluations in Section 4 of the research paper.
```
Comment (novaturient95) [inline assets/models/system/CodeLlama-13b-Python-hf/description.md:64]: update the positioning of samples. Its should be below

File: assets/models/system/CodeLlama-34b-Python-hf/description.md [hunk 6]
```diff
@@ -62,6 +62,19 @@ All experiments reported here and the released models have been trained and fine
 
 See evaluations for the main models and detailed ablations in Section 3 and safety evaluations in Section 4 of the research paper.
 
+# Model Evaluation samples
```
Comment (novaturient95) [inline assets/models/system/CodeLlama-34b-Python-hf/description.md:65]: Same as other comments

File: assets/models/system/CodeLlama-13b-Python-hf/description.md (hunk 5 above)
Comment (KonakanchiSwathi) [reply assets/models/system/CodeLlama-13b-Python-hf/description.md]: completed

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**